sklearn/tests/test_common.py

```diff
@@ -160,6 +159,9 @@ def test_configure():
             # The configuration spits out warnings when not finding
             # Blas/Atlas development headers
             warnings.simplefilter("ignore", UserWarning)
+            # TODO: remove the following filter once replace numpy.distutils
+            # by setuptools.
+            warnings.simplefilter("ignore", DeprecationWarning)
```

Comment:
This can go away if #24563 is merged first.

Reply:
#24563 has been merged, so this can be removed as you indicate:
```suggestion
            warnings.simplefilter("ignore", DeprecationWarning)
```